packages/storybook-html/stories/components/hero/hero.docs.mdx

```diff
@@ -8,84 +8,72 @@ import StatusBadge from "../../../.storybook/custom/components/StatusBadge";
 <StatusBadge.Development />
 <StatusBadge.DraftDoc />
 
-### Default hero
+## Overview
 
-<Canvas withSource="none" withToolbar>
-  <Story id="components-hero--default-hero" height={300} />
-</Canvas>
+The `.uq-hero` element provides the hero for homepage and level 1 and 2 landing
+pages.
 
-```html
-<div class="uq-hero">
-  <div class="uq-hero__content">
-    <div class="uq-hero__text">
-      <h1 class="uq-hero__title">Page title</h1>
-    </div>
-  </div>
-</div>
-```
+### How-to
+
+There are 4 elements within the hero.
+
+- Title: `h1.uq-hero__title`
+- Description: `.uq-hero__description` (it is optional include a `p` tag within the description)
+- Button (optional): `.uq-hero__button .uq-button.uq-button--purple`
+- Image: `.uq-hero__image`
 
-### Feature image hero
+There are 2 size variations - small (default) and large (use the `.uq-hero--large` modifier class).
 
-<Canvas withSource="none" withToolbar>
-  <Story id="components-hero--feature-hero" height={505} />
+There are 2 color variations - purple (default) and white (use the `.uq-hero--light` modifier class).
+
+<Canvas withSource="none">
+  <Story id="components-hero--hero" />
 </Canvas>
 
 ```html
-<div class="uq-hero uq-hero--feature-image">
+<div class="uq-hero uq-hero--light uq-hero--large">
   <div class="uq-hero__image">
-    <div class="uq-hero__image--small">
-      <img
-        src="/images/hero/hero-example--small.jpg"
-        alt=""
-        role="presentation"
-      />
-    </div>
-    <div class="uq-hero__image--large">
-      <img
-        src="/images/hero/hero-example--large.jpg"
-        alt=""
-        role="presentation"
-      />
-    </div>
+    <img src="/images/hero/hero-example-1.jpg" alt="" role="presentation" />
   </div>
-  <div class="uq-hero__content">
-    <div class="uq-hero__text">
-      <div class="uq-hero__title">Start postgraduate study online</div>
-      <p class="uq-hero__subtext">Apply by June 30</p>
-      <p><a class="uq-button" href="#">Learn more</a></p>
+  <div class="uq-container">
+    <div class="uq-hero__content">
```

Comment:
would these be better without the wrapping elements eg <p class="uq-hero__description">Description</p> and remove uq-hero__button?

Reply:
The wrappers provide surety around spacing without having to rely on the child elements - so it hopefully makes things a little more robust.